astropy/table/column.py

```diff
@@ -104,6 +103,23 @@ def __setslice__(self, start, stop, val):
             self.__setitem__(slice(start, stop), val)
 
 
+def _encode_str(value):
```

Comment:
I'd still put this on `BaseColumn` rather than as a separate function, so a possible subclass can override it.